src/lib/helpers/omit.ts

```diff
@@ -0,0 +1,10 @@
+export const omit = <T extends Record<string, unknown>, K extends keyof T>(obj: T, keys: string[]): Omit<T, K> =>
```

Comment:
I could not find `omit` or any useful helper library in the codebase (like `lodash`, `underscore` or `ramda`) so I just added this.